### DISCUSSION TITLE
Add project tabs so that all debug doesn't end up in the same pane ?

### DISCUSSION THREAD

**Discussion by vesper8** [spatie/ray#349] [General]
Right now I installed ray on multiple laravel local projects and also have it working with one remote project using `expose share http://localhost:23517`

I notice though that the debug logs don't differentiate between projects. At least the one that is being executed from a remote server shows the name of the remote server so I can tell it apart. But all the local ones don't give me a visual cue that they originated from different laravel projects.

I would much prefer if each project created its own tab with logs being sent to those individual tabs.. so all the logs would be project-specific and not all mixed and mingled together.

I really like how the tabs work on HELO in this regard. Once HELO receives new emails it automatically creates a new vertical tab. I hope this can of QoL enhancement comes to Ray in the future : )

**Comment by patinthehat** [comment]
This is essentially a duplicate of discussions #268 and #214, and possibly others.  
I'd also like to see this feature. :+1:

**Comment by vesper8** [comment]
I really hope someday this will be implemented.. either with tabs, groups, or at the least ability to target specific screens after creating a new screen
